### PR TITLE
small changes to the new section based on feedback

### PR DESCRIPTION
**Pull request by CZagrobelny**
This PR addresses feedback on the new section:
-   typo in the hash section ‘dictonary'
-  remove the MVC exercise which uses the server logs to walk through the request cycle
-  re-order the explanation of Model View Controller so that it flows more naturally with the request cycle explanation (Controller, Model, View)
-  Add pictures to help students understand link a file in sublime to a view in the browser

This was the low hanging fruit/essential things to change on the new section before we do another workshop.

I am still planning to submit PRs to the main repository with the new section as well as PRs for the other feedback we discussed....but with a workshop coming up in May, I wanted to make a PR with essential fixes to the new material.

I also feel the new section should be added to the intermediate tutorial for consistency, but will submit a PR seperately.

### CODE REVIEW

File: sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step [hunk 1]
```diff
@@ -22,6 +22,22 @@ explanation do
 
   message "Rails implements a very specific notion of the **Model/View/Controller** pattern, which guides how you structure your web applications."
 
+  h2 "Controller"
+  message <<-MARKDOWN
+  * Controllers pass Ruby objects between the Models and the Views.
```
Comment (maxjacobson) [inline sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step]: This line isn't new, so this shouldn't block this merge, but... do you think this line is as clear and correct as it ought to be? I'm trying to think of a suggested replacement. I think I would change this section to something like the following:

```
* When you visit any page in your application, that request will be handled by a method in a Controller.
* Controllers use the Models to create, read, update, and delete data from the database
* Controllers pass Ruby objects to the Views
```

File: sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step [hunk 2]
```diff
@@ -40,10 +56,20 @@ explanation do
...
+  message "**Let's take a look at the Topic View files in suggestotron and match each View file with the corresponding page in the browser.**"
+
+  message <<-MARKDOWN
+  The file **`app/views/topics/index.html.erb`** in your **text editor (Sublime)**:
+
+  ![Screenshot of topic index view file in Sublime](img/Topics_Index_File_Viewed_In_Sublime.png)
 
+  Creates this **page** in your **web browser**:
+
+  ![Screenshot of topic index page in a web browserread](img/Seattle_list_with_topic.png)
```
Comment (maxjacobson) [inline sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step]: nit: `browserread` -> `browser`

**Review by maxjacobson — APPROVED** [review]
this LGTM - thanks for remembering to do this follow-up work 👍

**Comment by qiheme** [comment]
@CZagrobelny can you follow up with @maxjacobson's comments?

File: sites/en/intro-to-rails/ruby_language.step [hunk 3]
```diff
@@ -68,7 +68,7 @@ fruits = fruits - ["kiwi"]
 
   step do
     console <<-RUBY
-fruit_dictonary[:strawberry]
+fruit_dictionary[:strawberry]
     RUBY
```
Comment (qiheme) [inline sites/en/intro-to-rails/ruby_language.step]: I think one of the students you guys taught that day made a PR to fix this already :-)

**Comment by qiheme** [comment]
Also, looks like this is just a restructure of what we have currently right?

File: sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step (hunk 1 above)
Comment (CZagrobelny) [reply sites/en/intro-to-rails/the_request_cycle_and_rails_architecture.step]: @maxjacobson I like your replacement, it's a difficult one to describe particularly before they have a solid understanding of Model and View.  Just pushed an update.

File: sites/en/intro-to-rails/ruby_language.step (hunk 3 above)
Comment (CZagrobelny) [reply sites/en/intro-to-rails/ruby_language.step]: That's awesome, pulled in master!

**Comment by CZagrobelny** [comment]
@qiheme @maxjacobson made changes in response to feedback.  And yes, the MVC section was essentially just re-ordered (also, I incorporated Max's suggested changes for the controller section).  Thanks!

**Review by qiheme — APPROVED** [review]

**Pull request merged**